resources/views/programas/partials/autores.blade.php

```diff
@@ -0,0 +1,15 @@
+@foreach($value['AUTORES'] as $k=>$val)
+                        
+    @if(strpos($val["NOME-PARA-CITACAO"], ';') !== false)
+        {!! explode(';', $val["NOME-PARA-CITACAO"])[0] !!}
+    @else 
+        {!! $val["NOME-PARA-CITACAO"] !!}  
+    @endif
+
+    @if( $k + 1 <  count($value['AUTORES']))
+        ;
```

Comment:
Se a condição for válida o que deve executar?

Reply:
Deve imprimir o ; (ponto e virgula)